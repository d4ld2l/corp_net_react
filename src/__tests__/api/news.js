import api from '../../api'

describe('API News', () => {
  it('should fetch all news', async () => {
    const response = await api.news.all()
    expect(response.data).toMatchSnapshot()
  })

  it('should find news by id', async () => {
    const response = await api.news.findById(35)
    expect(response.data).toMatchSnapshot()
  })

  it('should return news categories', async () => {
    const response = await api.news.categories()
    expect(response.data).toMatchSnapshot()
  })

  it('should return news by params', async () => {
    const response = await api.news.where({ params: { news_category_id: 6 } })
    expect(response.data).toMatchSnapshot()
  })
})
