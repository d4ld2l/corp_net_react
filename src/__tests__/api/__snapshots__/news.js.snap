// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`API News should fetch all news 1`] = `
Array [
  Object {
    "author": null,
    "body": "<p>16.10.2017 к нам присоединились</p><table><tbody id=\\"index\\"><tr><td><img src=\\"/assets/thumb_missing-f015378f0e113aa915f555c42fef4271455cce2836370ddbdb9c4b8c0a6d368e.png\\" alt=\\"Thumb missing\\" /></td><td>ну  как же  так</td><td></td><td></td></tr></tbody></table>",
    "comments": Array [],
    "comments_count": 0,
    "community_id": null,
    "created_at": "2017-10-17T07:00:16.147Z",
    "documents": Array [],
    "id": 35,
    "news_category": Object {
      "created_at": "2017-10-13T07:00:04.180Z",
      "id": 6,
      "name": "Новые коллеги",
      "updated_at": "2017-10-13T07:00:04.180Z",
    },
    "news_category_id": 6,
    "on_top": null,
    "photos": Array [],
    "preview": null,
    "published_at": null,
    "state": "published",
    "tags": Array [],
    "title": "Добро пожаловать!",
    "updated_at": "2017-10-17T07:00:16.147Z",
    "user_id": null,
  },
  Object {
    "author": null,
    "body": "<p>Сегодня 17.10.2017 родились</p><table><tbody id=\\"index\\"></tbody></table>",
    "comments": Array [],
    "comments_count": 0,
    "community_id": null,
    "created_at": "2017-10-17T07:00:16.143Z",
    "documents": Array [],
    "id": 34,
    "news_category": Object {
      "created_at": "2017-10-13T07:00:03.997Z",
      "id": 5,
      "name": "День рождения",
      "updated_at": "2017-10-13T07:00:03.997Z",
    },
    "news_category_id": 5,
    "on_top": null,
    "photos": Array [],
    "preview": null,
    "published_at": null,
    "state": "published",
    "tags": Array [],
    "title": "Поздравьте коллег",
    "updated_at": "2017-10-17T07:00:16.143Z",
    "user_id": null,
  },
  Object {
    "author": null,
    "body": "<p>15.10.2017 к нам присоединились</p><table><tbody id=\\"index\\"></tbody></table>",
    "comments": Array [],
    "comments_count": 0,
    "community_id": null,
    "created_at": "2017-10-16T07:00:02.975Z",
    "documents": Array [],
    "id": 33,
    "news_category": Object {
      "created_at": "2017-10-13T07:00:04.180Z",
      "id": 6,
      "name": "Новые коллеги",
      "updated_at": "2017-10-13T07:00:04.180Z",
    },
    "news_category_id": 6,
    "on_top": null,
    "photos": Array [],
    "preview": null,
    "published_at": null,
    "state": "published",
    "tags": Array [],
    "title": "Добро пожаловать!",
    "updated_at": "2017-10-16T07:00:02.975Z",
    "user_id": null,
  },
  Object {
    "author": null,
    "body": "<p>Сегодня 16.10.2017 родились</p><table><tbody id=\\"index\\"></tbody></table>",
    "comments": Array [],
    "comments_count": 0,
    "community_id": null,
    "created_at": "2017-10-16T07:00:02.968Z",
    "documents": Array [],
    "id": 32,
    "news_category": Object {
      "created_at": "2017-10-13T07:00:03.997Z",
      "id": 5,
      "name": "День рождения",
      "updated_at": "2017-10-13T07:00:03.997Z",
    },
    "news_category_id": 5,
    "on_top": null,
    "photos": Array [],
    "preview": null,
    "published_at": null,
    "state": "published",
    "tags": Array [],
    "title": "Поздравьте коллег",
    "updated_at": "2017-10-16T07:00:02.968Z",
    "user_id": null,
  },
  Object {
    "author": null,
    "body": "<p>14.10.2017 к нам присоединились</p><table><tbody id=\\"index\\"></tbody></table>",
    "comments": Array [],
    "comments_count": 0,
    "community_id": null,
    "created_at": "2017-10-15T07:00:30.573Z",
    "documents": Array [],
    "id": 31,
    "news_category": Object {
      "created_at": "2017-10-13T07:00:04.180Z",
      "id": 6,
      "name": "Новые коллеги",
      "updated_at": "2017-10-13T07:00:04.180Z",
    },
    "news_category_id": 6,
    "on_top": null,
    "photos": Array [],
    "preview": null,
    "published_at": null,
    "state": "published",
    "tags": Array [],
    "title": "Добро пожаловать!",
    "updated_at": "2017-10-15T07:00:30.573Z",
    "user_id": null,
  },
  Object {
    "author": null,
    "body": "<p>Сегодня 15.10.2017 родились</p><table><tbody id=\\"index\\"></tbody></table>",
    "comments": Array [],
    "comments_count": 0,
    "community_id": null,
    "created_at": "2017-10-15T07:00:30.569Z",
    "documents": Array [],
    "id": 30,
    "news_category": Object {
      "created_at": "2017-10-13T07:00:03.997Z",
      "id": 5,
      "name": "День рождения",
      "updated_at": "2017-10-13T07:00:03.997Z",
    },
    "news_category_id": 5,
    "on_top": null,
    "photos": Array [],
    "preview": null,
    "published_at": null,
    "state": "published",
    "tags": Array [],
    "title": "Поздравьте коллег",
    "updated_at": "2017-10-15T07:00:30.569Z",
    "user_id": null,
  },
  Object {
    "author": null,
    "body": "<p>13.10.2017 к нам присоединились</p><table><tbody id=\\"index\\"></tbody></table>",
    "comments": Array [],
    "comments_count": 0,
    "community_id": null,
    "created_at": "2017-10-14T07:00:32.625Z",
    "documents": Array [],
    "id": 29,
    "news_category": Object {
      "created_at": "2017-10-13T07:00:04.180Z",
      "id": 6,
      "name": "Новые коллеги",
      "updated_at": "2017-10-13T07:00:04.180Z",
    },
    "news_category_id": 6,
    "on_top": null,
    "photos": Array [],
    "preview": null,
    "published_at": null,
    "state": "published",
    "tags": Array [],
    "title": "Добро пожаловать!",
    "updated_at": "2017-10-14T07:00:32.625Z",
    "user_id": null,
  },
  Object {
    "author": null,
    "body": "<p>Сегодня 14.10.2017 родились</p><table><tbody id=\\"index\\"></tbody></table>",
    "comments": Array [],
    "comments_count": 0,
    "community_id": null,
    "created_at": "2017-10-14T07:00:32.608Z",
    "documents": Array [],
    "id": 28,
    "news_category": Object {
      "created_at": "2017-10-13T07:00:03.997Z",
      "id": 5,
      "name": "День рождения",
      "updated_at": "2017-10-13T07:00:03.997Z",
    },
    "news_category_id": 5,
    "on_top": null,
    "photos": Array [],
    "preview": null,
    "published_at": null,
    "state": "published",
    "tags": Array [],
    "title": "Поздравьте коллег",
    "updated_at": "2017-10-14T07:00:32.608Z",
    "user_id": null,
  },
  Object {
    "author": null,
    "body": "<p>12.10.2017 к нам присоединились</p><table><tbody id=\\"index\\"></tbody></table>",
    "comments": Array [],
    "comments_count": 0,
    "community_id": null,
    "created_at": "2017-10-13T07:00:04.722Z",
    "documents": Array [],
    "id": 27,
    "news_category": Object {
      "created_at": "2017-10-13T07:00:04.180Z",
      "id": 6,
      "name": "Новые коллеги",
      "updated_at": "2017-10-13T07:00:04.180Z",
    },
    "news_category_id": 6,
    "on_top": null,
    "photos": Array [],
    "preview": null,
    "published_at": null,
    "state": "published",
    "tags": Array [],
    "title": "Добро пожаловать!",
    "updated_at": "2017-10-13T07:00:04.722Z",
    "user_id": null,
  },
  Object {
    "author": Object {
      "birthday": "1989-02-28",
      "city": "Москва",
      "created_at": "2017-07-20T09:32:07.787Z",
      "email_private": "mishukov89@gmail.com",
      "fb_url": "https://www.facebook.com/dmitry.mishukov",
      "id": 3,
      "linkedin_url": "https://www.linkedin.com/in/dmitry-mishukov-56032668/",
      "middlename": "Михайлович",
      "name": "Дмитрий",
      "password": null,
      "phone_number_private": "+79652779137",
      "photo": Object {
        "for_community": Object {
          "url": "/uploads/profile/photo/3/for_community_2017-07-19_04.36.35.jpg",
        },
        "for_profile": Object {
          "url": "/uploads/profile/photo/3/for_profile_2017-07-19_04.36.35.jpg",
        },
        "thumb": Object {
          "url": "/uploads/profile/photo/3/thumb_2017-07-19_04.36.35.jpg",
        },
        "url": "/uploads/profile/photo/3/2017-07-19_04.36.35.jpg",
      },
      "sex": "male",
      "skype": "durnostay_w",
      "surname": "Мишуков",
      "telegram": "+79652779137",
      "updated_at": "2017-08-30T21:45:10.414Z",
      "user_id": 6,
      "vk_url": "https://vk.com/durnostay",
    },
    "body": "<p>Новые новости и вопросы вы можете задать в канале мессенджера innovation</p>
",
    "comments": Array [],
    "comments_count": 0,
    "community_id": null,
    "created_at": "2017-10-12T22:59:55.948Z",
    "documents": Array [],
    "id": 26,
    "news_category": Object {
      "created_at": "2017-08-01T18:09:58.331Z",
      "id": 1,
      "name": "Новости компании",
      "updated_at": "2017-08-01T18:09:58.331Z",
    },
    "news_category_id": 1,
    "on_top": false,
    "photos": Array [
      Object {
        "created_at": "2017-10-12T22:59:55.951Z",
        "file": Object {
          "for_community": Object {
            "url": "/uploads/photo/file/15/for_community_%D1%85%D0%BE%D0%BC%D0%B8%D1%87.jpg",
          },
          "for_profile": Object {
            "url": "/uploads/photo/file/15/for_profile_%D1%85%D0%BE%D0%BC%D0%B8%D1%87.jpg",
          },
          "thumb": Object {
            "url": "/uploads/photo/file/15/thumb_%D1%85%D0%BE%D0%BC%D0%B8%D1%87.jpg",
          },
          "url": "/uploads/photo/file/15/%D1%85%D0%BE%D0%BC%D0%B8%D1%87.jpg",
        },
        "id": 15,
        "name": "Игорь",
        "photo_attachable_id": 26,
        "photo_attachable_type": "NewsItem",
        "updated_at": "2017-10-12T22:59:55.951Z",
      },
    ],
    "preview": "Рубрика \\"Молния от Хомича\\"",
    "published_at": "2017-10-12T23:00:06.437Z",
    "state": "published",
    "tags": Array [
      Object {
        "id": 64,
        "name": "innovation",
        "taggings_count": 1,
      },
      Object {
        "id": 65,
        "name": "khomich",
        "taggings_count": 1,
      },
    ],
    "title": "News by Хомич",
    "updated_at": "2017-10-12T23:00:06.438Z",
    "user_id": 6,
  },
  Object {
    "author": Object {
      "birthday": "1989-02-28",
      "city": "Москва",
      "created_at": "2017-07-20T09:32:07.787Z",
      "email_private": "mishukov89@gmail.com",
      "fb_url": "https://www.facebook.com/dmitry.mishukov",
      "id": 3,
      "linkedin_url": "https://www.linkedin.com/in/dmitry-mishukov-56032668/",
      "middlename": "Михайлович",
      "name": "Дмитрий",
      "password": null,
      "phone_number_private": "+79652779137",
      "photo": Object {
        "for_community": Object {
          "url": "/uploads/profile/photo/3/for_community_2017-07-19_04.36.35.jpg",
        },
        "for_profile": Object {
          "url": "/uploads/profile/photo/3/for_profile_2017-07-19_04.36.35.jpg",
        },
        "thumb": Object {
          "url": "/uploads/profile/photo/3/thumb_2017-07-19_04.36.35.jpg",
        },
        "url": "/uploads/profile/photo/3/2017-07-19_04.36.35.jpg",
      },
      "sex": "male",
      "skype": "durnostay_w",
      "surname": "Мишуков",
      "telegram": "+79652779137",
      "updated_at": "2017-08-30T21:45:10.414Z",
      "user_id": 6,
      "vk_url": "https://vk.com/durnostay",
    },
    "body": "<p>500 дней до Универсиады</p>

<p><input alt=\\"\\" src=\\"/uploads/ckeditor/attachments/3/clock.png\\" style=\\"width: 720px; height: 720px;\\" type=\\"image\\" /></p>
",
    "comments": Array [],
    "comments_count": 0,
    "community_id": null,
    "created_at": "2017-10-02T10:02:25.992Z",
    "documents": Array [],
    "id": 23,
    "news_category": Object {
      "created_at": "2017-10-02T10:00:21.463Z",
      "id": 4,
      "name": "Универсиада 2019",
      "updated_at": "2017-10-02T10:00:21.463Z",
    },
    "news_category_id": 4,
    "on_top": false,
    "photos": Array [],
    "preview": "Скоро будет 500 дней до Универсиады",
    "published_at": "2017-10-02T10:02:42.489Z",
    "state": "published",
    "tags": Array [
      Object {
        "id": 60,
        "name": "uni2019",
        "taggings_count": 1,
      },
    ],
    "title": "500 дней до Универсиады",
    "updated_at": "2017-10-02T10:02:42.490Z",
    "user_id": 6,
  },
  Object {
    "author": Object {
      "birthday": "1989-02-28",
      "city": "Москва",
      "created_at": "2017-07-20T09:32:07.787Z",
      "email_private": "mishukov89@gmail.com",
      "fb_url": "https://www.facebook.com/dmitry.mishukov",
      "id": 3,
      "linkedin_url": "https://www.linkedin.com/in/dmitry-mishukov-56032668/",
      "middlename": "Михайлович",
      "name": "Дмитрий",
      "password": null,
      "phone_number_private": "+79652779137",
      "photo": Object {
        "for_community": Object {
          "url": "/uploads/profile/photo/3/for_community_2017-07-19_04.36.35.jpg",
        },
        "for_profile": Object {
          "url": "/uploads/profile/photo/3/for_profile_2017-07-19_04.36.35.jpg",
        },
        "thumb": Object {
          "url": "/uploads/profile/photo/3/thumb_2017-07-19_04.36.35.jpg",
        },
        "url": "/uploads/profile/photo/3/2017-07-19_04.36.35.jpg",
      },
      "sex": "male",
      "skype": "durnostay_w",
      "surname": "Мишуков",
      "telegram": "+79652779137",
      "updated_at": "2017-08-30T21:45:10.414Z",
      "user_id": 6,
      "vk_url": "https://vk.com/durnostay",
    },
    "body": "<p>По нашим планам к концу спринта будет готов следующий функционал</p>

<p>1. Главная страница</p>

<p>2. Список новостей</p>

<p>3. Карточка новости</p>

<p>4. Список пользователей</p>

<p>5. Профиль прользователя</p>
",
    "comments": Array [
      Object {
        "body": "Какой-то комментарий",
        "commentable_id": 10,
        "commentable_type": "NewsItem",
        "created_at": "2017-08-29T09:39:53.207Z",
        "id": 2,
        "parent_comment_id": null,
        "updated_at": "2017-08-29T09:39:53.207Z",
        "user": Object {
          "company_id": null,
          "created_at": "2017-07-20T09:30:55.980Z",
          "email": "dmishukov@fil-it.ru",
          "id": 6,
          "profile": Object {
            "birthday": "1989-02-28",
            "city": "Москва",
            "created_at": "2017-07-20T09:32:07.787Z",
            "email_private": "mishukov89@gmail.com",
            "fb_url": "https://www.facebook.com/dmitry.mishukov",
            "id": 3,
            "linkedin_url": "https://www.linkedin.com/in/dmitry-mishukov-56032668/",
            "middlename": "Михайлович",
            "name": "Дмитрий",
            "password": null,
            "phone_number_private": "+79652779137",
            "photo": Object {
              "for_community": Object {
                "url": "/uploads/profile/photo/3/for_community_2017-07-19_04.36.35.jpg",
              },
              "for_profile": Object {
                "url": "/uploads/profile/photo/3/for_profile_2017-07-19_04.36.35.jpg",
              },
              "thumb": Object {
                "url": "/uploads/profile/photo/3/thumb_2017-07-19_04.36.35.jpg",
              },
              "url": "/uploads/profile/photo/3/2017-07-19_04.36.35.jpg",
            },
            "sex": "male",
            "skype": "durnostay_w",
            "surname": "Мишуков",
            "telegram": "+79652779137",
            "updated_at": "2017-08-30T21:45:10.414Z",
            "user_id": 6,
            "vk_url": "https://vk.com/durnostay",
          },
          "provider": "email",
          "status": null,
          "uid": "dmishukov@fil-it.ru",
          "updated_at": "2017-10-17T18:18:18.141Z",
        },
        "user_id": 6,
      },
    ],
    "comments_count": 1,
    "community_id": null,
    "created_at": "2017-08-23T17:24:09.100Z",
    "documents": Array [],
    "id": 10,
    "news_category_id": null,
    "on_top": false,
    "photos": Array [
      Object {
        "created_at": "2017-08-28T09:10:15.262Z",
        "file": Object {
          "for_community": Object {
            "url": "/uploads/photo/file/2/for_community_%D0%A1%D0%BD%D0%B8%D0%BC%D0%BE%D0%BA_%D1%8D%D0%BA%D1%80%D0%B0%D0%BD%D0%B0_2017-08-28_%D0%B2_6.57.56.png",
          },
          "for_profile": Object {
            "url": "/uploads/photo/file/2/for_profile_%D0%A1%D0%BD%D0%B8%D0%BC%D0%BE%D0%BA_%D1%8D%D0%BA%D1%80%D0%B0%D0%BD%D0%B0_2017-08-28_%D0%B2_6.57.56.png",
          },
          "thumb": Object {
            "url": "/uploads/photo/file/2/thumb_%D0%A1%D0%BD%D0%B8%D0%BC%D0%BE%D0%BA_%D1%8D%D0%BA%D1%80%D0%B0%D0%BD%D0%B0_2017-08-28_%D0%B2_6.57.56.png",
          },
          "url": "/uploads/photo/file/2/%D0%A1%D0%BD%D0%B8%D0%BC%D0%BE%D0%BA_%D1%8D%D0%BA%D1%80%D0%B0%D0%BD%D0%B0_2017-08-28_%D0%B2_6.57.56.png",
        },
        "id": 2,
        "name": "Фото",
        "photo_attachable_id": 10,
        "photo_attachable_type": "NewsItem",
        "updated_at": "2017-08-28T09:10:15.262Z",
      },
      Object {
        "created_at": "2017-08-29T09:14:29.192Z",
        "file": Object {
          "for_community": Object {
            "url": "/uploads/photo/file/3/for_community_abe_simpson.png",
          },
          "for_profile": Object {
            "url": "/uploads/photo/file/3/for_profile_abe_simpson.png",
          },
          "thumb": Object {
            "url": "/uploads/photo/file/3/thumb_abe_simpson.png",
          },
          "url": "/uploads/photo/file/3/abe_simpson.png",
        },
        "id": 3,
        "name": "фото2",
        "photo_attachable_id": 10,
        "photo_attachable_type": "NewsItem",
        "updated_at": "2017-08-29T09:14:29.192Z",
      },
      Object {
        "created_at": "2017-08-29T09:14:29.207Z",
        "file": Object {
          "for_community": Object {
            "url": "/uploads/photo/file/4/for_community_burns.png",
          },
          "for_profile": Object {
            "url": "/uploads/photo/file/4/for_profile_burns.png",
          },
          "thumb": Object {
            "url": "/uploads/photo/file/4/thumb_burns.png",
          },
          "url": "/uploads/photo/file/4/burns.png",
        },
        "id": 4,
        "name": "фото3",
        "photo_attachable_id": 10,
        "photo_attachable_type": "NewsItem",
        "updated_at": "2017-08-29T09:14:29.207Z",
      },
    ],
    "preview": "До конца спринта MVP будет готов на 50%",
    "published_at": "2017-08-23T17:24:16.549Z",
    "state": "published",
    "tags": Array [
      Object {
        "id": 36,
        "name": "#shr",
        "taggings_count": 2,
      },
    ],
    "title": "Планы на спринт",
    "updated_at": "2017-08-28T09:08:59.152Z",
    "user_id": 6,
  },
  Object {
    "author": Object {
      "birthday": "1989-02-28",
      "city": "Москва",
      "created_at": "2017-07-20T09:32:07.787Z",
      "email_private": "mishukov89@gmail.com",
      "fb_url": "https://www.facebook.com/dmitry.mishukov",
      "id": 3,
      "linkedin_url": "https://www.linkedin.com/in/dmitry-mishukov-56032668/",
      "middlename": "Михайлович",
      "name": "Дмитрий",
      "password": null,
      "phone_number_private": "+79652779137",
      "photo": Object {
        "for_community": Object {
          "url": "/uploads/profile/photo/3/for_community_2017-07-19_04.36.35.jpg",
        },
        "for_profile": Object {
          "url": "/uploads/profile/photo/3/for_profile_2017-07-19_04.36.35.jpg",
        },
        "thumb": Object {
          "url": "/uploads/profile/photo/3/thumb_2017-07-19_04.36.35.jpg",
        },
        "url": "/uploads/profile/photo/3/2017-07-19_04.36.35.jpg",
      },
      "sex": "male",
      "skype": "durnostay_w",
      "surname": "Мишуков",
      "telegram": "+79652779137",
      "updated_at": "2017-08-30T21:45:10.414Z",
      "user_id": 6,
      "vk_url": "https://vk.com/durnostay",
    },
    "body": "<p>При публикации новости что-то идёт не так<input alt=\\"\\" src=\\"/uploads/ckeditor/attachments/1/abe_simpson.png\\" style=\\"width: 639px; height: 479px;\\" type=\\"image\\" /></p>

<p>И ещё что-то не так:&nbsp;<input alt=\\"\\" src=\\"/uploads/ckeditor/attachments/2/burns.png\\" type=\\"image\\" /></p>
",
    "comments": Array [],
    "comments_count": 0,
    "community_id": null,
    "created_at": "2017-08-23T11:06:55.062Z",
    "documents": Array [],
    "id": 9,
    "news_category": Object {
      "created_at": "2017-08-01T18:10:09.213Z",
      "id": 2,
      "name": "Новый интранет",
      "updated_at": "2017-08-01T18:10:09.213Z",
    },
    "news_category_id": 2,
    "on_top": false,
    "photos": Array [],
    "preview": "Проверка новостей групп",
    "published_at": "2017-08-25T06:21:56.354Z",
    "state": "published",
    "tags": Array [
      Object {
        "id": 52,
        "name": "новыйтег",
        "taggings_count": 1,
      },
    ],
    "title": "Что-то пошло не так",
    "updated_at": "2017-09-29T13:02:39.555Z",
    "user_id": 6,
  },
]
`;

exports[`API News should find news by id 1`] = `
Object {
  "body": "<p>16.10.2017 к нам присоединились</p><table><tbody id=\\"index\\"><tr><td><img src=\\"/assets/thumb_missing-f015378f0e113aa915f555c42fef4271455cce2836370ddbdb9c4b8c0a6d368e.png\\" alt=\\"Thumb missing\\" /></td><td>ну  как же  так</td><td></td><td></td></tr></tbody></table>",
  "comments": Array [],
  "comments_count": 0,
  "community_id": null,
  "created_at": "2017-10-17T07:00:16.147Z",
  "documents": Array [],
  "id": 35,
  "news_category": Object {
    "created_at": "2017-10-13T07:00:04.180Z",
    "id": 6,
    "name": "Новые коллеги",
    "updated_at": "2017-10-13T07:00:04.180Z",
  },
  "news_category_id": 6,
  "on_top": null,
  "photos": Array [],
  "preview": null,
  "published_at": null,
  "state": "published",
  "tags": Array [],
  "title": "Добро пожаловать!",
  "updated_at": "2017-10-17T07:00:16.147Z",
  "user_id": null,
}
`;

exports[`API News should return news by params 1`] = `
Array [
  Object {
    "author": null,
    "body": "<p>16.10.2017 к нам присоединились</p><table><tbody id=\\"index\\"><tr><td><img src=\\"/assets/thumb_missing-f015378f0e113aa915f555c42fef4271455cce2836370ddbdb9c4b8c0a6d368e.png\\" alt=\\"Thumb missing\\" /></td><td>ну  как же  так</td><td></td><td></td></tr></tbody></table>",
    "comments": Array [],
    "comments_count": 0,
    "community_id": null,
    "created_at": "2017-10-17T07:00:16.147Z",
    "documents": Array [],
    "id": 35,
    "news_category": Object {
      "created_at": "2017-10-13T07:00:04.180Z",
      "id": 6,
      "name": "Новые коллеги",
      "updated_at": "2017-10-13T07:00:04.180Z",
    },
    "news_category_id": 6,
    "on_top": null,
    "photos": Array [],
    "preview": null,
    "published_at": null,
    "state": "published",
    "tags": Array [],
    "title": "Добро пожаловать!",
    "updated_at": "2017-10-17T07:00:16.147Z",
    "user_id": null,
  },
  Object {
    "author": null,
    "body": "<p>15.10.2017 к нам присоединились</p><table><tbody id=\\"index\\"></tbody></table>",
    "comments": Array [],
    "comments_count": 0,
    "community_id": null,
    "created_at": "2017-10-16T07:00:02.975Z",
    "documents": Array [],
    "id": 33,
    "news_category": Object {
      "created_at": "2017-10-13T07:00:04.180Z",
      "id": 6,
      "name": "Новые коллеги",
      "updated_at": "2017-10-13T07:00:04.180Z",
    },
    "news_category_id": 6,
    "on_top": null,
    "photos": Array [],
    "preview": null,
    "published_at": null,
    "state": "published",
    "tags": Array [],
    "title": "Добро пожаловать!",
    "updated_at": "2017-10-16T07:00:02.975Z",
    "user_id": null,
  },
  Object {
    "author": null,
    "body": "<p>14.10.2017 к нам присоединились</p><table><tbody id=\\"index\\"></tbody></table>",
    "comments": Array [],
    "comments_count": 0,
    "community_id": null,
    "created_at": "2017-10-15T07:00:30.573Z",
    "documents": Array [],
    "id": 31,
    "news_category": Object {
      "created_at": "2017-10-13T07:00:04.180Z",
      "id": 6,
      "name": "Новые коллеги",
      "updated_at": "2017-10-13T07:00:04.180Z",
    },
    "news_category_id": 6,
    "on_top": null,
    "photos": Array [],
    "preview": null,
    "published_at": null,
    "state": "published",
    "tags": Array [],
    "title": "Добро пожаловать!",
    "updated_at": "2017-10-15T07:00:30.573Z",
    "user_id": null,
  },
  Object {
    "author": null,
    "body": "<p>13.10.2017 к нам присоединились</p><table><tbody id=\\"index\\"></tbody></table>",
    "comments": Array [],
    "comments_count": 0,
    "community_id": null,
    "created_at": "2017-10-14T07:00:32.625Z",
    "documents": Array [],
    "id": 29,
    "news_category": Object {
      "created_at": "2017-10-13T07:00:04.180Z",
      "id": 6,
      "name": "Новые коллеги",
      "updated_at": "2017-10-13T07:00:04.180Z",
    },
    "news_category_id": 6,
    "on_top": null,
    "photos": Array [],
    "preview": null,
    "published_at": null,
    "state": "published",
    "tags": Array [],
    "title": "Добро пожаловать!",
    "updated_at": "2017-10-14T07:00:32.625Z",
    "user_id": null,
  },
  Object {
    "author": null,
    "body": "<p>12.10.2017 к нам присоединились</p><table><tbody id=\\"index\\"></tbody></table>",
    "comments": Array [],
    "comments_count": 0,
    "community_id": null,
    "created_at": "2017-10-13T07:00:04.722Z",
    "documents": Array [],
    "id": 27,
    "news_category": Object {
      "created_at": "2017-10-13T07:00:04.180Z",
      "id": 6,
      "name": "Новые коллеги",
      "updated_at": "2017-10-13T07:00:04.180Z",
    },
    "news_category_id": 6,
    "on_top": null,
    "photos": Array [],
    "preview": null,
    "published_at": null,
    "state": "published",
    "tags": Array [],
    "title": "Добро пожаловать!",
    "updated_at": "2017-10-13T07:00:04.722Z",
    "user_id": null,
  },
]
`;

exports[`API News should return news categories 1`] = `
Array [
  Object {
    "created_at": "2017-08-01T18:09:58.331Z",
    "id": 1,
    "name": "Новости компании",
    "updated_at": "2017-08-01T18:09:58.331Z",
  },
  Object {
    "created_at": "2017-08-01T18:10:09.213Z",
    "id": 2,
    "name": "Новый интранет",
    "updated_at": "2017-08-01T18:10:09.213Z",
  },
  Object {
    "created_at": "2017-09-01T14:38:16.906Z",
    "id": 3,
    "name": "Повесть временных лет",
    "updated_at": "2017-09-01T14:38:16.906Z",
  },
  Object {
    "created_at": "2017-10-02T10:00:21.463Z",
    "id": 4,
    "name": "Универсиада 2019",
    "updated_at": "2017-10-02T10:00:21.463Z",
  },
  Object {
    "created_at": "2017-10-13T07:00:03.997Z",
    "id": 5,
    "name": "День рождения",
    "updated_at": "2017-10-13T07:00:03.997Z",
  },
  Object {
    "created_at": "2017-10-13T07:00:04.180Z",
    "id": 6,
    "name": "Новые коллеги",
    "updated_at": "2017-10-13T07:00:04.180Z",
  },
]
`;
